terraform/deployments/ecr/main.tf

```diff
@@ -176,3 +190,80 @@ resource "aws_iam_role_policy_attachment" "push_to_ecr_role_attachment" {
   role       = aws_iam_role.push_image_to_ecr_role.name
   policy_arn = aws_iam_policy.push_image_to_ecr_policy.arn
 }
+
+resource "aws_ecr_repository_policy" "pull_images_from_ecr_policy_policy" {
+  for_each   = toset(local.repositories)
+  repository = each.key
+  policy     = <<POLICY
```

Comment:
Could you change these to use `jsonencode()` please, so that Terraform can syntax-check them? (It's an easy change - literally all you need to do is write `jsonencode(...)` instead of `<<EOF ... EOF`, then run `terraform fmt` to keep the linter happy. There's an example [here](https://github.com/alphagov/govuk-infrastructure/compare/sengi/prometheus#diff-06308fdc5bf7909e591ecc888974eba1d8e36394b6a1068b4bbb845e1cb7028dR237) if you need.)

Reply:
will fix

Reply:
done in ecr and concourse-iam main files